classroom.rb

```diff
@@ -0,0 +1,13 @@
+class ClassRoom
+  attr_accessor :label, :students
```

Comment:
- Please declare students by using `attr_reader :students` because the method apply here is getter method